bin/mail/2017-01-20-outage/send-real-emails

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# 474630 teachers
+../send-to-mailing-list --limit 500000 2017-01-20-outage ./teachers.csv
```

Comment:
It's still a higher number, 478071, but not enough to increase the limit, using the DB.